src/main/java/gift/Product.java

```diff
@@ -0,0 +1,48 @@
+package gift;
+
+public class Product {
+
+    public Product() {}
+    public Product(Long id, String name, int price, String imageUrl) {
+        this.id = id;
+        this.name = name;
+        this.price = price;
+        this.imageUrl = imageUrl;
+    }
+    private Long id;
+    private String name;
+    private int price;
+    private String imageUrl;
```

Comment:
대부분 순서를 필드를 먼저 작성하고 생성자를 두긴 합니다.

```suggestion
    private Long id;
    private String name;
    private int price;
    private String imageUrl;
    
        public Product() {}
    public Product(Long id, String name, int price, String imageUrl) {
        this.id = id;
        this.name = name;
        this.price = price;
        this.imageUrl = imageUrl;
    }
```

이런식으로 말이죠~